tests/unit/test1_unit_tests/src/main.c

```diff
@@ -1108,14 +1111,15 @@ ZTEST(zcbor_unit_tests, test_pexpect)
 	zassert_true(zcbor_tag_pexpect(state_d, &(uint32_t){6}), NULL);
 	zassert_true(zcbor_simple_pexpect(state_d, &(uint8_t){7}), NULL);
 	zassert_true(zcbor_bool_pexpect(state_d, &(bool){true}), NULL);
+#ifndef CONFIG_BOARD_QEMU_MALTA_BE
```

Comment:
I assume the MALTA BE does not support float then? :D

Reply:
It's complicated :D. It doesn't support it with the default libc. I decided to fix it more completely in a separate PR.